js/controller.js

```diff
@@ -1362,9 +1362,22 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     showVolumeSliderBar: function() {
       this.state.volumeState.volumeSliderVisible = true;
+      if (Utils.isAndroid()) {
+        this.startHideVolumeSliderTimer();
+      }
       this.renderSkin();
     },
 
+    startHideVolumeSliderTimer: function() {
+        this.cancelTimer();
+        var timer = setTimeout(function() {
```

Comment:
This timer is different from the one discovery up next uses. The user might want to cancel the discovery timer explicitly as he may not want to see the next video auto-playing. But here the timer need not be cancelled explicitly as it starts once the slider becomes visible and it should auto hide as soon as the timer times out. The user intervention is not required here.